src/mira/client/mira_client.py

```diff
@@ -115,6 +116,9 @@ def deploy(self, flow: Flow):
             flow.metadata.author = flow.metadata.author[1:]
         return self.console.deploy_flow(flow.metadata.author, flow.metadata.name, flow.to_dict(), flow.version)
 
+    def get_yaml_flow(self, author_name: str, flow_name: str, version: Optional[str] = None) -> dict:
```

Comment:
Why are we creating a new method here? Doesn't client.flow.get does the same thing?